TestAppTypescript/ios/TestAppTypescript/Info.plist

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please git apply this patch so that we can test with push, had to this locally:

```
diff --git a/TestAppTypescript/ios/TestAppTypescript/Info.plist b/TestAppTypescript/ios/TestAppTypescript/Info.plist
index d11cea8..fff4790 100644
--- a/TestAppTypescript/ios/TestAppTypescript/Info.plist
+++ b/TestAppTypescript/ios/TestAppTypescript/Info.plist
@@ -22,6 +22,19 @@
        <string>????</string>
        <key>CFBundleVersion</key>
        <string>1</string>
+       <key>LSRequiresIPhoneOS</key>
+       <true/>
+       <key>NSAppTransportSecurity</key>
+       <dict>
+               <key>NSExceptionDomains</key>
+               <dict>
+                       <key>localhost</key>
+                       <dict>
+                               <key>NSExceptionAllowsInsecureHTTPLoads</key>
+                               <true/>
+                       </dict>
+               </dict>
+       </dict>
        <key>NSCameraUsageDescription</key>
        <string>$(PRODUCT_NAME) would like to use your camera</string>
        <key>NSLocationWhenInUseUsageDescription</key>
@@ -30,8 +43,10 @@
        <string>$(PRODUCT_NAME) would like to save photos to your photo gallery</string>
        <key>NSPhotoLibraryUsageDescription</key>
        <string>$(PRODUCT_NAME) would like access to your photo gallery</string>
-       <key>LSRequiresIPhoneOS</key>
-       <true/>
+       <key>UIBackgroundModes</key>
+       <array>
+               <string>remote-notification</string>
+       </array>
        <key>UILaunchStoryboardName</key>
        <string>LaunchScreen</string>
        <key>UIRequiredDeviceCapabilities</key>
@@ -46,19 +61,5 @@
        </array>
        <key>UIViewControllerBasedStatusBarAppearance</key>
        <false/>
-       <key>NSLocationWhenInUseUsageDescription</key>
-       <string></string>
-       <key>NSAppTransportSecurity</key>
-       <!--See http://ste.vn/2015/06/10/configuring-app-transport-security-ios-9-osx-10-11/ -->
-       <dict>
-               <key>NSExceptionDomains</key>
-               <dict>
-                       <key>localhost</key>
-                       <dict>
-                               <key>NSExceptionAllowsInsecureHTTPLoads</key>
-                               <true/>
-                       </dict>
-               </dict>
-       </dict>
 </dict>
 </plist>
```

Reply:
Corrupt at the end when applying here. (Corrupt patch at line 57).